### ISSUE TITLE
A better explanation about the n tensor 

### ISSUE THREAD

**Issue by celsofranssa** [rjagerman/pytorchltr#34]
The input to an LTR loss function comprises three tensors:

`scores`: a tensor of size (batch_size, number of scores)
`relevance`: a tensor of size (batch_size, number of relevance scores)
`n`:  a tensor of size (batch_size) ( the number of docs per learning instance)

Could you provide a better explanation of `n`?
